imagestreams/mariadb-centos.json

```diff
@@ -44,6 +44,42 @@
           "type": "Local"
         }
       },
+      {
+        "name": "10.11-c9s",
```

Comment:
name should be "10.11-el9"

Reply:
As you can see here: https://quay.io/organization/sclorg all our container that are part of CentOS Stream has the name either 'c9s' or 'c8s'

Reply:
And we do not want to rename all of them.

Reply:
Actually we were discussing this topic recently on team Slack. I was asking why also CentOS Stream imagestream tags are named -el9/-el8. Reason is, that "el" does not mean "rhel" and this is on purpose - to make the switch between CentOS Stream and RHEL imagestreams easier - and should not be changed.

Not really sure how the c8s/c9s tags got into mariadb container, but e.g., postgresql uses -el tags, as it should have. See [here](https://github.com/sclorg/postgresql-container/blob/master/imagestreams/postgresql-centos.json)

@hhorak do I understand this correctly please?